### ISSUE TITLE
Death from Sacrificial Dagger - No Grave, Death Compass points to area I didn't die at.

### ISSUE THREAD

**Issue by neonod** [DarkPacks/SevTech-Ages#2618]
New post for "Death from Sacrificial Dagger - No Grave, Death Compass points to area I didn't die at. #1620" since the closing remarks ask to have a new issue created if issue reoccurs.

## Bug Report
Death at blood magic altar by self sacrifice resulted in:
- no death log in chat
- death compass pointing to invalid location.
- no entry in tmg to perform a /tmg_restore with.
- no items left on ground.

## Expected Behaviour
When death occurs either leave items or a grave.

## Possible Solution
???
Possibly look into a command to recover last inventory? Number of bugs reported with missing/random location graves.

## Steps to Reproduce (for bugs)
1. stand in range of blood altar.
2. use sacrificial dagger too many times and die.

## Logs
* Client/Server Log: Client log - https://pastebin.com/9kVUWpRR
Server log - unavailable.
* Crash Log: No crash.

## Client Information
* Modpack Version: 3.0.7
* Java Version: 1.8.0.171
* Launcher Used: Twitch
* Memory Allocated: 8gb
* Server/LAN/Single Player: Server
* Optifine Installed: No
* Shaders Enabled: No

## Server Information
* Java Version: Unavailable
* Operating System: Linux
* Hoster/Hosting Solution: Private box - not a public hosting company.
* Sponge (Non-Vanilla Forge) Server: No
* Additional Content Installed: No

**Comment by artdude543** [comment]
You can use this command `/tmp_restore player {timestamp}` to restore your inventory. We are aware of the issues the Grave Mod is causing and considering our options for switching to a more stable one. But I'm going to close this as we have an internal chat in regards.

**Comment by neonod** [comment]
Please see my original comment in the issue report.... 

no entry in tmg to perform a /tmg_restore with.

in other words... /tmg_restore <name> <timestamp> no worky. 
and /tmg_restore <name> latest restores the last death saved in tmg which was over 2 weeks ago.

**Comment by artdude543** [comment]
TLDR: The grave mod we are using is ridiculous and causing way too many issues with regards to vanishing inventory. We have another one lined up, however. But we don't have a planned drop on it. So it's going have to be something to deal with for the time being. Sorry.
